packages/image/src/Image.story.js

```diff
@@ -69,14 +58,43 @@ storiesOf("Image", module)
       <Image src={imgSrc} badge size={badgeSizes.LARGE} />
     </div>
   ))
-  .add("fit (cover)", () => (
+  .add("custom dimensions", () => (
+    <Image src={imgSrc} width="800px" height="400px" />
+  ))
+  .add("fit (default/cover) - horizontal image", () => (
```

Comment:
What about changing these stories to group a set of props? Something similar to button and tooltip stories.

I guess we could have 
- 1 story covering thumbnail and its sizes
- 1 story covering badge and its sizes
- 1 story for fit and all its cases
- Other stories that don't fit the above cases